packages/shared/src/components/auth/AuthModal.tsx

```diff
@@ -49,6 +49,7 @@ export default function AuthModal({
       onRequestClose={onDiscardAttempt}
       className={classNames(styles.authModal, className)}
       contentClassName={classNames('auth', authVersion)}
+      style={{}}
```

Comment:
Since we introduced a default styling from one of the recent PRs, we need to override it since we already apply the styling in a dedicated css module in this case.